### PR TITLE
[v8] Migration guide for Web3Auth Modal SDK

### PR DESCRIPTION
**Pull request by AyushBherwani1998**

### CODE REVIEW

File: docs/pnp/migration-guides/modal-v7-to-v8.mdx [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+const web3auth = new Web3Auth({
+  clientId: "", // Get your Client ID from the Web3Auth Dashboard
+  web3AuthNetwork: "sapphire_mainnet",
+  chainConfig: {
+    ...chainConfig,
+    chainNamespace: CHAIN_NAMESPACES.EIP155,
+  },
+  uiConfig: {
+    // highlight-next-line
+    defaultLanguage: "tr", // en, de, ja, ko, zh, es, fr, pt, nl
```
Comment (rtomas) [inline docs/pnp/migration-guides/modal-v7-to-v8.mdx:75]: You can change text “en” to type.
import { LANGUAGES } from "@toruslabs/openlogin-utils";
LANGUAGES.tr

File: docs/pnp/migration-guides/modal-v7-to-v8.mdx [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    chainId: "0x3",
+    displayName: "Solana Testnet",
+    chainNamespace: CHAIN_NAMESPACES.SOLANA,
+    tickerName: "SOLANA",
+    ticker: "SOL",
+    decimals: 18,
+    rpcTarget: "https://api.testnet.solana.com",
+    blockExplorerUrl: "https://explorer.solana.com/?cluster=testnet",
+    logo: "https://images.toruswallet.io/sol.svg"
+};
```
Comment (rtomas) [inline docs/pnp/migration-guides/modal-v7-to-v8.mdx:154]: isTestnet:true

File: docs/pnp/migration-guides/modal-v7-to-v8.mdx (hunk 1 above)
Comment (AyushBherwani1998) [reply docs/pnp/migration-guides/modal-v7-to-v8.mdx:75]: Done

File: docs/pnp/migration-guides/modal-v7-to-v8.mdx (hunk 2 above)
Comment (AyushBherwani1998) [reply docs/pnp/migration-guides/modal-v7-to-v8.mdx:154]: Done

**Pull request merged**